provider/storage_benchmark.py

```diff
@@ -222,12 +244,18 @@ def clean(self, timeout=1800, force=False):
                       by force, otherwise wait they are done
         :type force: bool
         """
+        # In order to the output of the previous session object does not
+        # disturb the current session object to get the shell prompt.
```

Comment:
```
4. Re-create a new session in the method clean in order to the output of the
previous session object does not disturb the current session object to get
the shell prompt.
```
Could you explain more about this ?   I did not get the point, create a new session, do nothing, then close it.

Reply:
if the guest do session('dd if=/dev/zero of=/dev/sda bs=1M count=1024 &') by background, then do the next sesssion, it wanna get the prompt will be failed since the the previous session output will print to the current session.

e.g:
[root@ibm-p8-kvm-02-qe kar]# dd if=/dev/urandom of=/tmp/test bs=1M count=1024 &
[2] 70827
[1]   Done                    dd if=/dev/zero of=/tmp/test bs=1M count=10
[root@ibm-p8-kvm-02-qe kar]# 1024+0 records in
1024+0 records out
1073741824 bytes (1.1 GB, 1.0 GiB) copied, 5.23549 s, 205 MB/s

[2]+  Done                    dd if=/dev/urandom of=/tmp/test bs=1M count=1024
[root@ibm-p8-kvm-02-qe kar]#